mgr/src/server/server.cr

```diff
@@ -113,21 +113,25 @@ module StorageMgr
 
     # Fetch all the Services that belong to this node
     services = [] of MoanaTypes::ServiceUnit
+
     if GlobalConfig.agent
-      # TODO: Handle Authentication
-      # TODO: Get Server URL from GlobalConfig.local_node
-      # url = "http://localhost:3000/api/v1/pools/#{GlobalConfig.local_node.pool_name}/nodes/#{GlobalConfig.local_node.name}/services"
-      # resp = HTTP::Client.get(url)
-      # # TODO: Exit on error
-      # if resp.status_code == 200
-      #   services = Array(MoanaTypes::ServiceUnit).from_json(resp.body)
-      # end
-      Log.debug &.emit("Fetch services of this node")
-    else
-      services = Datastore.list_services(
-        GlobalConfig.local_node.pool_name,
-        GlobalConfig.local_node.name
-      )
+      loop do
+        begin
+          # TODO: Handle Authentication
+          # TODO: Get Server URL from GlobalConfig.local_node
+          url = "http://localhost:#{Kemal.config.port}/api/v1/pools/#{GlobalConfig.local_node.pool_name}/nodes/#{GlobalConfig.local_node.name}/services"
```

Comment:
Change the URL to use mgr url. Construct the URL like this https://github.com/kadalu/moana/blob/main/mgr/src/server/routes.cr#L35-L41

Reply:
Proxy will not work here as discussed previously since the local server is not started yet. So use the Mgr URL as above comment